modules/desktop/graphics/ghaf-launcher.nix

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{pkgs, ...}: let
```

Comment:
This is actually a definition of a package, not a module.

If this is called via pkgs.callPackage, then this line can be replaced with just `{writeShellScriptBin}: let` ...